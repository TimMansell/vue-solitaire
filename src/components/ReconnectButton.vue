<template>
  <Button :click="reconnect" data-test="reconnect-btn"> Reconnect </Button>
</template>

<script>
import Button from './Button.vue';

export default {
  name: 'ReconnectButton',
  components: {
    Button,
  },
  methods: {
    reconnect() {
      window.location.href = '/';
    },
  },
};
</script>
