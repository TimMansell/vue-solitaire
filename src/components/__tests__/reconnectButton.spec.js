import { shallowMount } from '@vue/test-utils';
import ReconnectButton from '@/components/ReconnectButton.vue';

describe('ReconnectButton.vue', () => {
  it('renders the component without crashing', () => {
    const wrapper = shallowMount(ReconnectButton);

    expect(wrapper.isVisible()).toBe(true);
  });
});
