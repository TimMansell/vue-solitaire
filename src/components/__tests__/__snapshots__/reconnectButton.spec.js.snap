// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReconnectButton.vue matches snapshot 1`] = `<button-stub type="default" size="md" click="[Function]" route="" data-test="reconnect-btn"> Reconnect </button-stub>`;
